SU-HSDO/suhumsci
# SHS-91: Codeception: Test video embeds and captions

# READY FOR REVIEW

## Summary
Adds a test to verify that video embeds can be added with a caption and that captions show up on the public view.

## Urgency
low

## Steps to Test
1. Run codeception tests either via CI or locally
2. Verify that the testVideoEmbed test passes.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
